soh/soh/Enhancements/randomizer/3drando/logic.cpp

```diff
@@ -580,6 +581,7 @@ namespace Logic {
     CanOpenStormGrotto  = CanPlay(SongOfStorms) && (ShardOfAgony || LogicGrottosWithoutAgony);
     HookshotOrBoomerang = CanUse(HOOKSHOT) || CanUse(BOOMERANG);
     CanGetNightTimeGS   = (CanPlay(SunsSong) || !NightGSExpectSuns);
+    CanBreakBeehives    = HookshotOrBoomerang || CanUse(BOW);
```

Comment:
I dont believe Bow can break beehives unless you changed that. When I have the time I'll also try and PR adding the beehive bombchu trick to ur branch unless you'd like to do that yourself

Reply:
I don't remember where I got the information but yeah, bow can't actually break beehives. I'll fix that.
What is the beehive bombchu trick? Is it just blowing the beehives up with bombchus?

Reply:
Yea but its considered a trick in logic (so will end up looking like `CanBreakBeehives = HookshotOrBoomerang || (LogicBombchuBeehives && HasBombchus)` or whatever the name is for logical chus)

Reply:
Fixed the bow problem and added the trick (also included bombs in the trick because they can also break the beehives).